Description: Modify setup.py to not install to /usr/local/share/
Author: Daniel Lintott <daniel@serverb.co.uk>
Forwarded: Not needed, debian specific
---
--- a/setup.py
+++ b/setup.py
@@ -283,8 +283,7 @@
                 'GNS3.Ui.ConfigurationPages',
                 'GNS3.Langs'],
           package_data = { 'GNS3': ['Langs/*.qm', 'Dynagen/configspec'] },
-          data_files = [ (wrapper_dir, ['qemuwrapper/qemuwrapper.py', 'vboxwrapper/vboxcontroller_4_1.py', 'vboxwrapper/vboxwrapper.py', 'vboxwrapper/tcp_pipe_proxy.py']),
-                        ('/usr/local/share/examples/gns3/', ['baseconfig.txt', 'baseconfig_sw.txt']),
-                        ('/usr/local/share/doc/gns3/', ['README', 'COPYING', 'CHANGELOG']),
-                        ('/usr/local/share/man/man1/', ['docs/man/gns3.1'])]
+          data_files = [ (wrapper_dir, ['qemuwrapper/qemuwrapper.py', 'vboxwrapper/vboxcontroller_4_1.py', 'vboxwrapper/vboxwrapper.py', 'vboxwrapper/tcp_pipe_proxy.py',
+                                        'baseconfig.txt', 'baseconfig_sw.txt']),
+                         ('/usr/share/man/man1/', ['docs/man/gns3.1'])]
     )
